candy-machine/program/src/lib.rs

```diff
@@ -125,7 +124,7 @@ pub mod candy_machine {
                     &payer.key(),
                     &gatekeeper.gatekeeper_network,
                     None,
-                )?;
+                );
```

Comment:
```suggestion
                )?;
```

Need to add that `?` back in.